src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
+
+  &__images {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: $font-size-m;
+    line-height: $line-h-l;
+    margin: 0 0 4px;
+    transition: color $time-for-animation;
+  }
+
+  &:hover &__name {
+    color: #34568b;
+  }
+
+  &__code-product {
+    font-weight: 400;
+    font-size: $font-size-s;
+    line-height: $line-h-s;
+    margin: 0 0 16px;
+    color: $second-color;
```

Comment:
The class name '__code-product' seems to be inconsistent with BEM naming convention. If it's an element of the 'product-card' block, it should be named '__code' instead. Verify that the class name follows the BEM naming convention as outlined in the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).